index.html

```diff
@@ -458,6 +458,31 @@ <h2><dfn>Applying the restriction transformation</dfn></h2>
           to <var>intersection</var>.
         </li>
       </ol>
+      <p>
+        Let the <dfn>decorated bounding box</dfn> of an element be smallest
```

Comment:
Maybe better to directly cite this definition directly in CSS Images Module 4?

Reply:
Either works for me.

Reply:
Changed to a citation, to reduce redundancy between specs.